From: Stained Glass OS contributors
Subject: winedbg: a program's crash is told in this system's words

The crash dialog blamed "a deficiency in Wine" and linked WineHQ's
application database and bug tracker, and the debugger retitled the crashed
program's console "Wine Debugger". Users of Stained Glass OS are not Wine's
users to send there.

--- a/programs/winedbg/winedbg.c
+++ b/programs/winedbg/winedbg.c
@@ -607,7 +607,8 @@
     SetConsoleCtrlHandler(ctrl_c_handler, TRUE);
 
     /* set our own title */
-    SetConsoleTitleA("Wine Debugger");
+    /* Stained Glass OS: the crashed program's console may be the one shown */
+    SetConsoleTitleA("Debugger");
 }
 
 static int dbg_winedbg_usage(BOOL advanced)
--- a/programs/winedbg/winedbg.rc
+++ b/programs/winedbg/winedbg.rc
@@ -35,8 +35,8 @@
 STRINGTABLE
 BEGIN
     IDS_AUTO_CAPTION     "Program Error"
-    IDS_INVALID_PARAMS   "A program on your system has crashed, but WineDbg was \
-                         unable to attach to the process to obtain a backtrace."
+    IDS_INVALID_PARAMS   "A program on your system has stopped working, and \
+                         the details of what happened could not be collected."
     IDS_UNIDENTIFIED     "(unidentified)"
     IDS_TEXT_FILES,      "Text files (*.txt)"
     IDS_ALL_FILES,       "All files (*.*)"
@@ -53,9 +53,8 @@
     LTEXT           "The program %s has encountered a serious problem and needs \
                     to close. We are sorry for the inconvenience.",
                     IDC_STATIC_TXT1,27,10,224,30
-    CONTROL         "This can be caused by a problem in the program or a deficiency in Wine. \
-                    You may want to check the <a href=\"https://appdb.winehq.org\">Application Database</a> \
-                    for tips about running this application.",
+    CONTROL         "This can be caused by a problem in the program, or in how Stained Glass OS \
+                    runs it. Show Details has what happened, to save or pass on.",
                     IDC_STATIC_TXT2,"SysLink",0,27,60,224,100
     DEFPUSHBUTTON   "Close", IDOK, 205, 151, 60, 16, WS_TABSTOP
     PUSHBUTTON      "Show &Details", ID_DETAILS, 140, 151, 60, 16, WS_TABSTOP
@@ -67,10 +66,8 @@
 FONT 8, "MS Shell Dlg"
 BEGIN
     EDITTEXT        IDC_CRASH_TXT, 4, 4, 392, 272, ES_MULTILINE | ES_READONLY | WS_HSCROLL | WS_VSCROLL, WS_TABSTOP
-    CONTROL         "If this problem has not been reported yet, \
-                     you can save the detailed information to a file using the \"Save As\" button, \
-                     then <a href=\"https://gitlab.winehq.org/wine/wine/-/wikis/Bugs\">file a bug report</a> \
-                     and attach that file to the report.",
+    CONTROL         "To report this problem, save these details to a file with \"Save As\" \
+                     and attach the file to the report.",
                     IDC_STATIC_TXT2,"SysLink",0,6,285,388,32
     DEFPUSHBUTTON   "Close", IDOK, 326, 320, 70, 16, WS_TABSTOP
     PUSHBUTTON      "&Save As...", ID_SAVEAS, 252, 320, 70, 16, WS_TABSTOP
